.github/project-labels.yml

```diff
@@ -0,0 +1,142 @@
+- name: category:bug
+  color: "5319e7"
+  description: >
+    The Issue/PR describes or solves a perceived malfunction within the game.
+- name: category:feature
+  color: "5319e7"
+  description: The Issue/PR describes or implements a new game feature.
+- name: category:refactoring
+  color: "5319e7"
+  description: The Issue/PR describes or contains an improved implementation.
+- name: category:tweak
+  color: "5319e7"
+  description: >
+    The PR contains insignificant code changes, like code style grooming or
+    value tweaking.
+- name: component:AI
+  color: "d93f0b"
+  description: >
+    The Issue/PR deals with the Python AI decision making code or affects it.
+- name: component:art music text assets
+  color: "d93f0b"
+  description: >
+    The Issue/PR deals with creating, modifying or verifying game assets.
+- name: component:build system
+  color: "d93f0b"
+  description: >
+    The Issue/PR deals with CMake, Visual Studio or the general build process.
+- name: component:content scripting
+  color: "d93f0b"
+  description: >
+    The Issue/PR deals with the FOCS language, turn events or the universe
+    generator.
+- name: component:deployment
+  color: "d93f0b"
+  description: >
+    The Issue/PR deals with the deployment process or distribution packages.
+- name: component:game mechanic
+  color: "d93f0b"
+  description: >
+    The Issue/PR deals with the currently used or planned game mechanics.
+- name: component:infrastructure
+  color: "d93f0b"
+  description: >
+    The Issue/PR deals with development aiding infrastructure like CI,
+    distribution or helper tools.
+- name: component:internal
+  color: "d93f0b"
+  description: >
+    The Issue/PR deals with any project component that has no explicit
+    `component` label.
+- name: component:network
+  color: "d93f0b"
+  desscription: >
+    The Issue/PR deals with TCP/IP networking and establishing a test network
+    infrastructure.
+- name: component:technical docs
+  color: "d93f0b"
+  description: >
+    The Issue/PR deals with technical documentation like API, style, workflow
+    or similar documentation.
+- name: component:translation
+  color: "d93f0b"
+  description: >
+    The Issue/PR deals with the game terminology and the translation process
+    in general.
+- name: component:UI
+  color: "d93f0b"
+  description: >
+    The Issue/PR deals with the game interface, a consistent representation
```

Comment:
Confused by `a consistent representation style in-game`

Reply:
Not sure what is meant by that, tbh.

Reply:
You don't know what I meant, or don't know what the description means?

Maybe `The Issue/PR deals with the game user interface, graphical or other`

Reply:
I'm talking about the label description itself.  The `consistent representation` part does not fit into the listing. Going with your suggestion.